### PR TITLE
New User Profile Field A/B Test

### PR DESCRIPTION
**Pull request by biyeun**
@calellowitz cc @orangejenny

### CODE REVIEW

File: corehq/apps/hqwebapp/templates/hqwebapp/crispy/radioselect.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  </li>
+  <li class="radio-select_items">
+    <input class="radio-select_expand" type="radio" name="{{ field.html_name }}" id="id_{{ field.html_name }}_opener" />
+    <label class="radio-select_closeLabel" for="id_{{ field.html_name }}_close"></label>
+
+    <ul class="radio-select_options">
+    {% for choice in field.field.choices %}
+      <li class="radio-select_option">
+        <input class="radio-select_input"
+               type="radio"{% if choice.0|stringformat:"s" == field.value|stringformat:"s" %}
```
Comment (orangejenny) [inline corehq/apps/hqwebapp/templates/hqwebapp/crispy/radioselect.html]: curious what this `choice.0` is, I'm unfamiliar

Comment (orangejenny) [reply corehq/apps/hqwebapp/templates/hqwebapp/crispy/radioselect.html]: also, nitpick, but the formatting of this if statement is hard to read

Comment (orangejenny) [reply corehq/apps/hqwebapp/templates/hqwebapp/crispy/radioselect.html]: nevermind, figured out this is how you handle tuples in templates.

**Comment by orangejenny** [comment]
Just to be very sure, enterprise projects don't use the signup workflow, right? Or is there already logic in the A/B test code that only runs them for SaaS?

File: corehq/apps/registration/views.py [hunk 2]
```diff
@@ -170,6 +171,7 @@ class UserRegistrationView(NewUserNumberAbTestMixin, BasePageView):
 
     @use_jquery_ui
     @use_ko_validation
+    @use_new_analytics
```
Comment (orangejenny) [inline corehq/apps/registration/views.py]: Wasn't there going to be qa of the new analytics code? I understand wanting to use the new code, just curious.

File: corehq/apps/registration/views.py [hunk 3]
```diff
@@ -210,12 +225,14 @@ def page_context(self):
...
                 initial=prefills,
                 show_number=self.ab_show_number,
+                show_persona=self.ab_show_persona,
             ),
             'reg_form_defaults': prefills,
             'hide_password_feedback': settings.ENABLE_DRACONIAN_SECURITY_FEATURES,
             'implement_password_obfuscation': settings.OBFUSCATE_PASSWORD_FOR_NIC_COMPLIANCE,
             'show_number': self.ab_show_number,
             'ab_show_number': self.ab_show_number_context,
+            'ab_persona_field': self.ab_persona_field.context,
```
Comment (orangejenny) [inline corehq/apps/registration/views.py]: Just from a readability / copy-and-pasting code perspective, it's weird that this is `ab_persona_field.context` but the line above is `ab_show_number_context `

**Review by orangejenny — APPROVED** [review]

File: corehq/apps/registration/views.py (hunk 2 above)
Comment (biyeun) [reply corehq/apps/registration/views.py]: yeah, I did some local QA of this and fixed a bug. The registration view is pretty siloed from everything else.

**Comment by biyeun** [comment]
> Just to be very sure, enterprise projects don't use the signup workflow, right? Or is there already logic in the A/B test code that only runs them for SaaS?

I'm very sure they don't. You have to be invited to the project space. The phone number field shows up in the enterprise signup if that's the case, as there's no logic behind blocking that.

**Comment by sheelio** [comment]
@biyeun 
What exactly is this change?

**Comment by biyeun** [comment]
@sheelio 
It A/B tests adding a drop down to the new user registration form to ask the use case for what the user plans to do with commcare. Marshall and Dev have been leading this one

**Pull request merged**